### PR TITLE
Fix duplicated contents in README.md

### PR DESCRIPTION
**Pull request by itugs**
### What does this PR do?
* Remove duplicated section
* Change search keyword to fit the screenshot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,13 +68,7 @@ If you use Apollo, let it use your `OkHttpClient`, and you'll get RUM and APM in
...
 ## Looking up your logs
 
-When you open your console in Datadog, navigate to the Logs section. In the search bar, type `source:android`. This filters your logs to only show the ones coming from mobile applications (Android and iOS).
-
-![Datadog Mobile Logs](docs/images/screenshot.png)
-
-## Looking up your logs
-
-When you open your console in Datadog, navigate to the Logs section. In the search bar, type `source:android`. This filters your logs to only show the ones coming from mobile applications (Android and iOS).
+When you open your console in Datadog, navigate to the Logs section. In the search bar, type `source:mobile`. This filters your logs to only show the ones coming from mobile applications (Android and iOS).
```
Comment (xgouchet) [inline README.md:71]: Actually the screenshot is wrong, but thanks a lot for pointing to us this duplication !

Comment (xgouchet) [reply README.md:71]: We'll fix that immediately

**Review by xgouchet — CHANGES_REQUESTED** [review]

**Comment by xgouchet** [comment]
A new PR (#440) has been made to fix this and add missing documentation and screenshots. 

Thanks again @itugs for bringing our attention on this topic.

**Comment by itugs** [comment]
@xgouchet glad I could help :)

**Pull request closed without merging**